clickplc/driver.py

```diff
@@ -73,15 +115,19 @@ async def set(self, address, data):
 
         >>> plc.set('df1', 0.0)  # Sets DF1 to 0.0
         >>> plc.set('df1', [0.0, 0.0, 0.0])  # Sets DF1-DF3 to 0.0.
-        >>> plc.set('y101', True)  # Sets Y101 to true
+        >>> plc.set('AV-101', True)  # Sets address nicknamed AV-101 to true
```

Comment:
Ambiguity edit - 

```python
plc.set('myTagName', True)  # Sets coil tagged as "myTagName"
```

Remove AV-101 because it looks like y101. Rename "nicknamed" to "tagged".

Reply:
Same here, I don't know if it's confusing for users for us to use tags when the software says nicknames

Reply:
If it says nicknames everywhere then we match, `myNickname` with the original inline comment